index.js

```diff
@@ -65,6 +89,9 @@ function fetchPageOfDataAndFilter(url) {
                     .then((json) => {
                         let filteredForImportant = filterResponseForImportantEvents(json);
                         importantEvents = importantEvents.concat(filteredForImportant);
+                        if (/^true|True$/.test(process.env.IGNORE_OWN_EVENTS)) {
```

Comment:
Since we're allowing both true and True, let's also allow TRUE and any other version of the word true that a user might put in. 
An easy way to do that would be to use the javascript function `toLowerCase()` instead of RegEx, but you could also write a more inclusive RegEx.